lib/std/math/big/int.zig

```diff
@@ -2817,7 +2818,7 @@ pub const Managed = struct {
     /// Converts self to a string in the requested base. Memory is allocated from the provided
     /// allocator and not the one present in self.
     pub fn toString(self: Managed, allocator: Allocator, base: u8, case: std.fmt.Case) ![]u8 {
-        if (base < 2 or base > 16) return error.InvalidBase;
+        if (base < 2 or base > 36) return error.InvalidBase;
```

Comment:
This seems an odd choice as invalid parameters should ideally be an assertion and only guarded against outside of this method. 

The original interface had the check however so maybe it's not to be fixed here.

Reply:
Yes this should be an assertion but that should be left for a different PR.